src/lib-components/AlphabeticalBrowseBy.vue

```diff
@@ -111,25 +111,21 @@ export default {
     computed: {
         parsedAlphabet: {
             get() {
-                let letterClass = "letter "
+                let letterClass = "letter"
```

Comment:
I don't think this line is being used, since we have the same line inside the return.